site/docs/theming/index.mdx

```diff
@@ -0,0 +1,113 @@
+
+While a user could change the status foreground color for `Toast`, this wouldn't change the corresponding color for the `Dialog` component. Changing the color associated with the `--salt-status-error-foreground` token ensures that both components, and any others that refer to the token, all use the same color in the given context.
+
+<ExampleContainer type="positive" customPillText="Best practice">
+  When theming, we recommend modifying tokens rather than applying tweaks to
+  each individual component.
+</ExampleContainer>
+
+### Benefits of tokens
+
+Tokens offer several benefits:
+
+- Streamlined design changes that automatically apply to related components
+- Increased consistency across all components, improving the overall user experience
+- Simplified approach when adding new components or patterns
+- Single reference points for commonly-used values
+
+### Which tokens to modify
+
+Salt is under active development, and future versions may modify the token list. Some tokens, however, are less likely to change than others.
+
+Salt's tokens are organized into three tiers. From most generalized to most specific, the tiers are:
+
+1. Foundational tokens
+2. Palette tokens
+3. Characteristic tokens
+
+The characteristic tokens are likely to experience the fewest changes.
+
+<ExampleContainer type="positive" customPillText="Best practice">
+  At this stage in Salt's development, we recommend modifying characteristic
+  tokens.
+</ExampleContainer>
+
+## Token tiers explained
+
+Salt's token tiers indicate the purpose of a token and how it is used in the design system. Tokens can reference each other and inherit values, so a token from one tier can provide information to another.
+
+Take the status characteristic tokens as an example. Components like [`Toast`](/salt/components/toast/examples) use these tokens to denote status using color, such as a green foreground color to denote success.
+
+1. The component uses the `--salt-status-success-foreground` characteristic token. This token has the default value `var(--salt-palette-success-foreground)`.
+2. The characteristic token references the palette token `--salt-palette-success-foreground`. This token has the default value `var(--salt-color-green-400)`.
+3. The palette token references the foundational token `--salt-color-green-400`. This token has the default value `rgb(48, 156, 90)`.
+
+Characteristics are both light/dark and density sensitive. That means the value the characteristic points to will change according to the values given in the nearest `SaltProvider`.
+
+As an example, `--salt-actionable-primary-background` resolves to `var(--salt-palette-interact-primary-background)`. In the light theme, this palette token has the value `var(--salt-color-gray-60)`. In the dark theme, the value is `var(--salt-color-gray-300)`. Foundational tokens then resolve to their respective RGBA values.
```

Comment:
Simplify the last part? "The RGBA value of these color tokens is defined in the Foundational layer".